### PR TITLE
Add more UIToolkit Mapping (weather, cards)

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
We want to remove the UIToolkit less files from our projects. The LESS there was intended for standalone open source projects and the design of elements don't fit with our brands.
Previously we have mapped elements with bootstrap like-for-like with toolkit components. This PR add the styling for components which aren't present in bootstrap. Brand new components

#### What tests does this PR have?
None

#### How can this be tested?
I have another branch in the pattern library with the same name.
If you `npm link` pattern library to this brands repo, then navigate to `molecules` -> `Credit/Debit Cards`
You should see some nice credit / debit card logos.

AND/OR

You can boot up master of pattern library and view the planner page and see if the weather icons work correctly.

Any questions give me a shout

#### What domains are affected?
None

#### Screenshots / Screencast
screen shot 2016-05-10 at 15 21 48

#### What gif best describes how you feel about this work?
Into the Map

---

**Notes when reviewing Static Site Generator projects, please think carefully about what can be done to centralise common functionality. Please also pay attention to anything configurable that will also need updating in the Yeoman generator template**

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2**
- [x] :+1:

**Review 3**
- [x] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

### CODE REVIEW

**Comment by stuartgiles** [comment]
Taking 1

File: brands/globals/src/less/toolkit/components/_payment-card.less [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*------------------------------------*\
+    #Card
+\*------------------------------------*/
+
+// Default size
+@cardWidth: 52px;
```
Comment (stuartgiles) [inline brands/globals/src/less/toolkit/components/_payment-card.less]: Variables should be declared in a seperate _variables.less file. This makes it possible to overwrite the variable in a theme. If the variable is declared in the same file that applies it, it's impossible to overwrite, which defeats the object.

File: brands/globals/src/less/toolkit/components/_payment-card.less [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+@cardHeight: 32px;
+
+.component-payment-card {
+  background:url('//d2r3bolva79zkl.cloudfront.net/assets/ui-toolkit/cards.png') no-repeat;
```
Comment (stuartgiles) [inline brands/globals/src/less/toolkit/components/_payment-card.less]: Potential variable opportunity for asset-path

File: brands/globals/src/less/toolkit/components/_weather.less [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+       url('//d2r3bolva79zkl.cloudfront.net/assets/ui-toolkit/fonts/meteocons-webfont.ttf') format('truetype'),
+       url('//d2r3bolva79zkl.cloudfront.net/assets/ui-toolkit/fonts/meteocons-webfont.svg#MeteoconsRegular') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+.component-weather {
+  width:auto;
+  text-align:center;
+  width:100px;
```
Comment (stuartgiles) [inline brands/globals/src/less/toolkit/components/_weather.less]: Potential variable opportunity

File: brands/globals/src/less/toolkit/components/_weather.less [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    border-bottom:0;
+  }
+}
+
+.weather-variant(@weather) {
+  padding-bottom:@padding-small-vertical;
+  &:before {
+    font-family: 'MeteoconsRegular';
+    content: "@{weather}";
+    font-size: 3.5em;
```
Comment (stuartgiles) [inline brands/globals/src/less/toolkit/components/_weather.less]: Potential variable opportunity

**Comment by stuartgiles** [comment]
I've tested both Credit/Debit Cards and Weather, both work for me.
I've added a few comments.
Adding a seperate _variables.less file is important for theming.
The other comments are just nice to haves.

File: brands/globals/src/less/toolkit/components/_payment-card.less (hunk 2 above)
Comment (jodiedoubleday) [inline brands/globals/src/less/toolkit/components/_payment-card.less]: Although this is a variable, I'm not sure card logos should be a variable we change by brand. I think card logos should be the same size regardless of brand.

File: brands/globals/src/less/toolkit/components/_payment-card.less (hunk 1 above)
Comment (jodiedoubleday) [inline brands/globals/src/less/toolkit/components/_payment-card.less]: Although this is a variable, I'm not sure card logos should be a variable we change by brand. I think card logos should be the same size regardless of brand.

**Comment by hxtomprice** [comment]
Taking :two:

**Comment by hxtomprice** [comment]
Looks good :+1:

**Comment by DuncanFenning** [comment]
Looks like I'm on 3️⃣

**Comment by DuncanFenning** [comment]
Looks good 👍 Weather and cards checked

**Pull request merged**